admission-webhook/webhook/base/cluster-role.yaml

```diff
@@ -15,3 +15,59 @@ rules:
   - create
   - patch
   - delete
+
```

Comment:
Editors shouldn't have permission to change poddefaults.
Those are reserved for cluster admin who setup like cloud credentials.

Reply:
Thanks for providing this context @kunmingg, I'll address it.